Libraries/xcformatter/Sources/NullFormatter.cpp

```diff
@@ -0,0 +1,130 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#include <xcformatter/NullFormatter.h>
+#include <pbxbuild/Tool/Invocation.h>
+#include <pbxbuild/Build/Context.h>
+
+#include <algorithm>
+
+using xcformatter::NullFormatter;
+
+NullFormatter::
```

Comment:
Probably not, I think this is because I copied it from the `DefaultFormatter` which has a constructor argument.